spicy/toolchain/src/compiler/codegen/unit-builder.cc

```diff
@@ -201,13 +201,15 @@ Type CodeGen::compileUnit(const type::Unit& unit, bool declare_only) {
     }
 
     if ( unit.usesRandomAccess() ) {
+        auto attr_random_access = Attribute("&requires_type", builder::string("uses_random_access"));
```

Comment:
Is `&requires_type` the right name here? The previous patch calls these "features", so would `&requires_feature` be more consistent?

Reply:
Went with `&requires_type_feature` as discussed offline.